src/sonic-yang-mgmt/sonic_yang.py

```diff
@@ -50,7 +51,18 @@ def __init__(self, yang_dir, debug=False):
     def __del__(self):
         pass
 
+    def disable_logging(self):
+        ly.set_log_options(0)
+        self.syslogging_disabled = True
+
+    def enable_logging(self):
+        # ly.LY_LOLOG: If callback is set use it, otherwise just print. If flag is not set, do nothing.
+        ly.set_log_options(ly.LY_LOLOG)
```

Comment:
The default flags were:
`LY_LOLOG | LY_LOSTORE_LAST`

LY_LOSTORE_LAST: means only the last error is stored and can be accessed using `ly.get_ly_errors(ctx)`. In `sonic_yang` we don't use `get_ly_errors(ctx)` so there is no need to store the last errors.

Reply:
[reference](https://netopeer.liberouter.org/doc/libyang/devel/html/group__log.html#ga9a039b617d81acfe361f641e5a4f8618)